inc/Engine/Optimization/RUCSS/Controller/UsedCSS.php

```diff
@@ -245,13 +254,19 @@ public function treeshake( string $html ): string {
 			return $html;
 		}
 
-		if ( 'completed' !== $used_css->status || empty( $used_css->css ) ) {
+		if ( 'completed' !== $used_css->status || empty( $used_css->hash ) ) {
+			return $html;
+		}
+
+		$used_css_content = $this->filesystem->get_used_css( $used_css->hash );
+
+		if ( empty( $used_css_content ) ) {
```

Comment:
do we need to delete the DB row it as a safeguard to make all rows synced with the DB?

Reply:
I think that's a good idea